shared/constants/types/teams.js

```diff
@@ -26,8 +26,9 @@ export type _PublicitySettings = {
   team: boolean,
 }
 
-export type _TeamSettings = RPCTypes.TeamSettings
```

Comment:
`RPCTypes.TeamSettings` is read-only, which causes some problems with making a record type out of it

Reply:
You can try spreading it

Reply:
The spreading may be confusing flow and making it bail on typing _TeamSettings...will investigate.

Reply:
Fixed this -- turns out we need `{...$Exact<Foo>}` as otherwise the keys will become optional